tests/test_optimizer.py

```diff
@@ -983,6 +983,97 @@ def test_constrained_AL_scalar():
     np.testing.assert_array_less(-Dwell, ctol)
 
 
+@pytest.mark.slow
+@pytest.mark.unit
+@pytest.mark.optimize
+def test_signed_PlasmaVesselDistance():
```

Comment:
how slow is this? does it need to be a unit test?

Reply:
about 20 seconds, I can make it a a regression test

Reply:
yes